py3-cffi.yaml

```
@@ -0,0 +1,43 @@
+# Generated from https://pypi.org/project/cffi/
+package:
+  name: py3-cffi
+  version: 1.15.1
+  epoch: 0
+  description: Foreign Function Interface for Python calling C code.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - py3-pycparser
+      - python-3.11
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3.11
+      - python-3.11-dev
+      - py3.11-setuptools
+      - libffi-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: d400bfb9a37b1351253cb402671cea7e89bdecc294e8016a707f6d1d8ac934f9
+      uri: https://files.pythonhosted.org/packages/2b/a8/050ab4f0c3d4c1b8aaa805f70e26e84d0e27004907c5b8ecc1d31815f92a/cffi-${{package.version}}.tar.gz
```

I think we can switch this to remove the sha and auto upgrades should work
```suggestion
      uri: https://foss.heptapod.net/pypy/cffi/-/archive/v${{package.version}}/cffi-v${{package.version}}.tar.gz
```

---

thank you!